etc/adminhtml/system.xml

```diff
@@ -18,6 +18,11 @@
                         <source_model>Phpro\Scheduler\Config\Source\TimelineLimit</source_model>
                         <comment><![CDATA[Limit the number of entries on the timeline page]]></comment>
                     </field>
+                    <field id="limit_successful_jobs" translate="label comment" type="select" sortOrder="30" showInDefault="1">
+                        <label>Limit successful jobs in timeline</label>
```

Comment:
Magento likes to start each word with a capital in the configuration labels. Personally I thinks it's neater if you do the same. So 'Limit successful jobs in timeline' would become 'Limit Successful Jobs In Timeline'.